transcripts/share-apis/orgs/org-add-members.json

```diff
@@ -2,22 +2,22 @@
   "body": {
     "members": [
       {
-        "avatarUrl": null,
-        "handle": "newuser",
-        "name": null,
-        "userId": "U-<UUID>"
+        "roles": "org_maintainer",
+        "subject": {
+          "avatarUrl": null,
+          "handle": "test",
+          "name": null,
+          "userId": "U-<UUID>"
+        }
       },
       {
-        "avatarUrl": null,
-        "handle": "test",
-        "name": null,
-        "userId": "U-<UUID>"
-      },
-      {
-        "avatarUrl": "https://www.gravatar.com/avatar/205e460b479e2e5b48aec07710c08d50?f=y&d=retro",
-        "handle": "transcripts",
-        "name": "The Transcript User",
-        "userId": "U-<UUID>"
+        "roles": "org_owner",
+        "subject": {
+          "avatarUrl": "https://www.gravatar.com/avatar/205e460b479e2e5b48aec07710c08d50?f=y&d=retro",
+          "handle": "transcripts",
+          "name": "The Transcript User",
+          "userId": "U-<UUID>"
+        }
       }
     ]
   },
```

Comment:
Here's the new response for the 'members' endpoint, it includes the user's role.